generators/server/templates/src/main/java/package/domain/_Authority.java

```diff
@@ -34,8 +34,10 @@
  * An authority (a security role) used by Spring Security.
  */<% if (databaseType === 'sql') { %>
 @Entity
-@Table(name = "jhi_authority")<% if (hibernateCache !== 'no') { %>
-@Cache(usage = CacheConcurrencyStrategy.NONSTRICT_READ_WRITE)<% } %><% } %><% if (databaseType === 'mongodb') { %>
+@Table(name = "jhi_authority")
+<%_ if (hibernateCache !== 'no') { if (hibernateCache === 'infinispan') { _%>
+@Cache(usage = CacheConcurrencyStrategy.READ_WRITE) <%_ } else { _%>
```

Comment:
Why don't use "NONSTRICT_READ_WRITE" like we usually do? I know it's a tradeoff between speed and consistency, but at least we should have the same with every option